services/minespace-web/src/styles/components/Overview.scss

```diff
@@ -46,3 +46,25 @@
     }
   }
 }
+
+.work-information-container {
+  border: 1px solid $gov-grey;
+  border-radius: 5px;
+  padding: 10px 20px;
+}
+
```

Comment:
All the classes listed below should be moved to layout.scss 

.right, .margin-small, .margin-large, .margin-xlarge